django-backend/fecfiler/transactions/schedule_a/serializers.py

```diff
@@ -70,6 +70,20 @@ def create(self, validated_data: dict):
             for child in children:
                 child["parent_transaction_object_id"] = parent.id
                 self.create(child)
+
+            # If this is a PARTNERSHIP_MEMO being created, update the parent
+            # contribution_purpose_description which contains text that
+            # depends on whether the parent PARTNERSHIP_RECEIPT has child
+            # transactions
+            partnership_receipt_cpd = "See Partnership Attribution(s) below"
+            if parent.transaction_type_identifier == "PARTNERSHIP_MEMO":
+                grandparent = ScheduleATransaction.objects.get(
```

Comment:
You'll need to refer to the Transactions model or ScheduleA model now with the 316 code that went into develop.

I notice that we've split 371(parent-child updates) functionality between the front ent (parent updates child) and back end (child updates parent).  Is it possible to not split the updating brain across repos?

If we want to keep it client side, we could just have the client fetch the parent before saving and post the updates together.